code/game/turfs/closed/_closed.dm

```diff
@@ -12,7 +12,7 @@
 
 /turf/closed/Initialize(mapload)
 	. = ..()
-	if(use_splitvis)
+	if(use_splitvis && (smoothing_flags & SMOOTH_BITMASK))
```

Comment:
what's she for?

Reply:
that element has 
`if(!(target_atom.smoothing_flags & SMOOTH_BITMASK))
		CRASH("We tried to splitvis [target.type] without bitmask smoothing. What?")
`
and some turf didnt have bitmasked smoothing, so it was runtiming a lot

Reply:
some turfs ought to runtime, because they need this implemented/that flag set and it hasn't been. 
Implementation is more a matter of having the sprites but if it's got use_splitvis and no flag then that's a bug

Reply:
```suggestion
	if(use_splitvis)
```